### ISSUE TITLE
Investigate what's blocking main process in simple-stacking

### ISSUE THREAD

**Issue by jrwbabylonlab** [babylonlabs-io/simple-staking#425]
![Image](https://github.com/user-attachments/assets/d5260b86-bc17-4156-a16a-16202dd1f781)

When the page loads, the main process is obvisouly been blocked and doing some heavy work. Figure out what it is and find a way to mitigate it.

**Comment by supertong** [comment]
I don't think this is valid anymore. The production build now only takes around 2-3 seconds on my laptop to finish loading.
